llvm/lib/Target/ARM/ARMSelectionDAGInfo.cpp

```diff
@@ -35,6 +35,19 @@ bool ARMSelectionDAGInfo::isTargetMemoryOpcode(unsigned Opcode) const {
          Opcode <= ARMISD::LAST_MEMORY_OPCODE;
 }
 
+static bool isAEABIFunctionImpl(const TargetLowering &TLI, RTLIB::Libcall LC) {
```

Comment:
I think this function name is misleading; a quick grep shows there are many more AEABI functions (IIUC). This being limited to a few memory functions is based on the early exit below its callsite.